modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -5,41 +5,47 @@ endif::[]
 
 This example sends a message with metadata to an Amazon SQS queue and then receives it from the queue. The example is split into the two flows:
 
-. The first flow sends a message with metadata. Then it gets the count of the messages in the queue to validate that the message was sent.
-. The second flow receives the message and logs the message body.
-
 image::amazon/amazon-sqs-message-flow.png[sqs-message-flow]
 
-This example uses variables for some field values. You can either replace the variables with their values in the code, or you can provide the values for each variable in the `src/main/resources/mule-artifact.properties` file.
+. The first flow sends a message with metadata and then gets the count of the messages in the queue to validate that the message was sent.
+//  LIST ITEMS SHOULD BE ONE SENTENCE MAX. Additional statements, if necessary, belong in a paragraph beneath the item.  GLOBAL.  //
+. The second flow receives the message and logs the message body.
+
+This example uses variables for some field values. You can either replace the variables with actual values in the code, or you can provide the actual values for each variable in the `src/main/resources/mule-artifact.properties` file.
 
-Before you try the example, access Anypoint Studio, and check whether the Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors.
+Before you try the example, access Anypoint Studio and verify that Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors before trying this example.
+//  BECAUSE THIS IS A PREREQUISTE TO THE ENTIRE EXAMPLE, consider moving this text to the very beginning of this introduction, or creating an actual Prerequisites subheading.  //
 
 == Create a Flow to Send a Message
 
-Start the first flow by sending a message to the queue:
+Start the example by creating a flow that sends a message to the queue. Creating the flow involves creating a new Mule project and configuring the Listener operation, adding a transform message, adding and configuring an SQS Send Message operation, adding a Logger component to display the response, obtaining the number of messages in the queue, and adding a Logger to display that number.
+//  I SUGGEST ADDING THIS TEXT TO PREPARE THE AUDIENCE FOR THE SUBTOPICS WITHIN THIS CONTAINER TOPIC. This is one of the GLOBAL requirements for a topic set intro text, along with addressing the audience as "you" and telling "you" exactly WHY they want or need the information in this topic set.  //
```

Comment:
```suggestion
```

Reply:
I SUGGEST ADDING THIS TEXT TO PREPARE THE AUDIENCE FOR THE SUBTOPICS WITHIN THIS CONTAINER TOPIC. This is one of the GLOBAL requirements for a topic set intro text, along with addressing the audience as "you" and telling "you" exactly WHY they want or need the information in this topic set.